lib/tools/adb-commands.js

```diff
@@ -250,10 +251,10 @@ methods.setDataState = async function (on) {
 methods.setWifiAndData = async function ({wifi:w, data:d}) {
```

Comment:
@scottdixon let's fix the styling on this line here (even though it wasn't part of your change)